pacman_vision: {
    cropping: true,
    downsampling: false,
    downsampling_leaf_size: 0.01,
    keep_organized: false,
    limit_xmax: 0.5,
    limit_xmin: -0.5,
    limit_ymax: 0.5,
    limit_ymin: -0.5,
    limit_zmax: 1.5,
    limit_zmin: 0.3,
    publish_limits: true,
    segmenting: false,
    plane_tolerance: 0.005,
    estimator: {
        cluster_tol: 0.05,
        iterations: 5,
        neighbors: 20,
        object_calibration: false,
        always_success: true,
        rmse_thresh: 0.005},
    sensor: {
        internal: false,
        name: "kinect2_optical_frame",
        topic: "/camera/depth_registered/points"}
}
